.github/workflows/ci.yml

```diff
@@ -47,6 +47,44 @@ jobs:
       - name: run Cargo clippy
         run: cargo clippy -- -D warnings -A deprecated
 
+  android:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - nightly
+    env:
+      CARGO_TERM_COLOR: always
+      # 20 MiB stack
+      RUST_MIN_STACK: 20971520
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      - name: Install Rust
+        run: |
+          rustup set auto-self-update disable
+          rustup toolchain install ${{ matrix.rust }} --profile minimal --component rustfmt clippy
+          rustup target add aarch64-linux-android
+          echo CARGO_TERM_COLOR=always >> $GITHUB_ENV
+          echo CARGO_INCREMENTAL=0 >> $GITHUB_ENV
+          echo RUST_BACKTRACE=1 >> $GITHUB_ENV
```

Comment:
```suggestion
```

These should just be moved into the `env` section

Reply:
yeah, probably, I just copied the previous job for now